app/models/customer.rb

```diff
@@ -20,4 +20,9 @@ def number_of_successful_transactions
     .where("result = ?", "0")
     .count
   end
+
+  def name
+    first_name + " " + last_name
+  end
```

Comment:
we added a method to ApplicationHelper that does this, since we considered it to be presentation logic